peoplescoaches/unix-shell
# issue 2 - create shell prompt

What's Changed:
- Makefile has additional commands to add the unix-shell directory to your PATH. this way you'll be able to run the shell command anywhere in your machine!
- the shell (called pcshell for people's coaches shell) has an initial lil prompt!
- the make command 'exec' has just been turned into 'compile' so that `pcshell` can be called and executed outside of the make command.

To Test: 
- Pull down this branch, run `make clean` and whichever setup command you need (`make setup_bash` or `make setup_zsh`). 
- Run `make compile`. 
- Run `pcshell`. Verify that the prompt appears and you can type in things.
- Try running `pcshell` in a directory other than the unix-shell directory to verify that the PATH was set up correctly.
